api/join_group.js

```diff
@@ -0,0 +1,19 @@
+exports.joinGroup = joinGroup
+
+function joinGroup (token) {
+  let invite
+  try {
+    invite = require('../src/db/group_invitations').byToken(token)
+  } catch (Error) {
+    return { success: false, error: 'invalidToken' }
+  }
+  try {
+    require('../src/db/user_in_group').addUser(invite.userId, invite.groupId)
```

Comment:
As far as I can tell this does not require any authentication from `invite.userId` (security!)